packages/tldraw/src/lib/shapes/geo/GeoShapeUtil.tsx

```diff
@@ -465,7 +465,8 @@ export class GeoShapeUtil extends BaseBoxShapeUtil<TLGeoShape> {
 						}}
 					>
 						<TextLabel
-							id={id}
+							id={id + '-label'}
+							shapeId={id}
```

Comment:
We probably don't want to add this one as the `SVGContainer` sibling already sets the id.

Reply:
> We probably don't want to add this one as the `SVGContainer` sibling already sets the id.

In this case, do we even need to do this PR? If a user can target this with JS / CSS already?

Reply:
Not sure I follow, it will be <div class="tl-shape"><svg id={id}/><div id={id + '-label'}/></div>